js/index.js

```diff
@@ -14,9 +14,64 @@ function bindToForm(form) {
 	update()
 
 	form.addEventListener('change', update)
-	form.addEventListener('keyup', update)
+	form.addEventListener('keyup', function(e) {
+		handleDynamicForms(e, update)
```

Comment:
Why remap handler only for `keyup`? This is brittle. Make `update` a dispatcher that calls both `handleDynamicForms` and rename the current `update` to some name that makes the difference with `handleDynamicForms` clear.